packages/atomic/src/components/result-template-components-v1/atomic-result-sections/atomic-result-section-badges.tsx

```diff
@@ -2,14 +2,13 @@ import {Element, Component} from '@stencil/core';
 import {containsVisualElement} from '../../../utils/utils';
 
 /**
- * The badges section helps the information seekers understand at a glance
- * special features about the items.
+ * This section provides badges that highlight special features of the item.
```

Comment:
I feel like this description implies that the section will automatically generate badges; what it actually does is just position them.

Would something like this be good?
```suggestion
* This section should contain badges that highlight special features of the item.
```